### ISSUE TITLE
Logo component need an editable alt attribute value

### ISSUE THREAD

**Issue by js-think-modular** [bigbluebutton/greenlight#6266]
The logo component is breaking accessibility (WCAG 2.2 AA SC 1.1.1 and SC 1.4.5) and is not editable for setting the correct alt-attribute value.

From SC 1.4.5:
"Logotypes (text that is part of a logo or brand name) are considered essential."

Generally the context in understanding there is: We have no control how the logo would be therefore it is a matter to set the alt text cleanly and explicit.
 
Proposal: The upload form of the Logo should also have an input field to set the correct alt-attribute value (it should be required).
 
The alt text is currently hard-coded with the value "CompanyLogo".
https://github.com/bigbluebutton/greenlight/blob/master/app/javascript/components/shared_components/Logo.jsx
